Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:jessie
+MAINTAINER projectcalico
+
+RUN mkdir -p /opt/cni/bin
+ADD ./dist/calico /opt/cni/bin
+ADD ./dist/calico-ipam /opt/cni/bin
+ENV PATH=$PATH:/opt/cni/bin
+WORKDIR /opt/cni/bin
```

Comment:
It would be nice if you added VOLUME /opt/cni so that a hyperkube container could just include --volumes-from calico-cni in its runtime params